src/notebook_TempCorr.ipynb

```diff
@@ -0,0 +1,7188 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "        <script type=\"text/javascript\">\n",
+       "        window.PlotlyConfig = {MathJaxConfig: 'local'};\n",
+       "        if (window.MathJax) {MathJax.Hub.Config({SVG: {font: \"STIX-Web\"}});}\n",
+       "        if (typeof require !== 'undefined') {\n",
+       "        require.undef(\"plotly\");\n",
+       "        requirejs.config({\n",
+       "            paths: {\n",
+       "                'plotly': ['https://cdn.plot.ly/plotly-latest.min']\n",
+       "            }\n",
+       "        });\n",
+       "        require(['plotly'], function(Plotly) {\n",
+       "            window._Plotly = Plotly;\n",
+       "        });\n",
+       "        }\n",
+       "        </script>\n",
+       "        "
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# PYTHON CODE\n",
+    "# Module imports\n",
+    "\n",
+    "from scipy import interpolate\n",
+    "import matplotlib.pyplot as plt\n",
+    "import plotly.plotly as py\n",
```

Comment:
Plotly isn't currently in the [requirements.txt](https://github.com/rrsg2020/analysis/blob/master/requirements.txt) file for the project, so it will need to be added + a version rule.

Reply:
@mathieuboudreau I added plotly version 3.10.0 (the one used for the qMRLab blogpost notebooks)